packages/connect-ui/src/components/BottomRightFloatingBar.tsx

```diff
@@ -34,13 +32,16 @@ type BottomRightFloatingBarProps = {
 };
 
 export const BottomRightFloatingBar = ({ onAnalyticsConfirm }: BottomRightFloatingBarProps) => {
-    const [view, setView] = useState(getView());
+    const initialStorage = storage.load();
+    const [view, setView] = useState(getView(initialStorage.tracking_enabled === undefined));
+    const isInitialTrackingEnabled = !!initialStorage.tracking_enabled;
```

Comment:
```suggestion
    const isInitialTrackingEnabled = initialStorage.tracking_enabled !== false;
```
To keep it on by default

Reply:
DONE https://github.com/trezor/trezor-suite/pull/11299/commits/12c47e46aba2f76d613115129d003b0119657c7c